identity/app/services/MembersDataApiService.scala

```diff
@@ -0,0 +1,62 @@
+package services
+
+import common.{HttpStatusException, Logging}
+import play.api.libs.ws.{DefaultWSCookie, WSClient, WSCookie}
+import play.api.mvc.{Cookie, Cookies}
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import scala.concurrent.{ExecutionContext, Future}
+
+// Modeled on members-data-api Attributes - ConsentAccess
+// https://github.com/guardian/members-data-api/blob/master/membership-attribute-service/app/models/Attributes.scala
+case class ContentAccess(
+  isMember: Boolean, // TODO a friend membership should not prevent auto-account deletion but currently does
+  isPaidMember: Boolean,
+  isRecurringContributor: Boolean,
+  hasDigitalPack: Boolean,
+  isPaperSubscriber: Boolean,
+  isGuardianWeeklySubscriber: Boolean
+) {
+  def canProceedWithAutoDeletion: Boolean = !(isMember || isPaidMember || isRecurringContributor || hasDigitalPack || isPaperSubscriber || isGuardianWeeklySubscriber)
+  def hasPaidProducts: Boolean = isPaidMember || isRecurringContributor || hasDigitalPack || isPaperSubscriber || isGuardianWeeklySubscriber
+}
+
+object ContentAccess {
+  implicit val jsRead: Reads[ContentAccess] = (
+    (JsPath \ "member").read[Boolean] and
+      (JsPath \ "paidMember").read[Boolean] and
+      (JsPath \ "recurringContributor").read[Boolean] and
+      (JsPath \ "digitalPack").read[Boolean] and
+      (JsPath \ "paperSubscriber").read[Boolean] and
+      (JsPath \ "guardianWeeklySubscriber").read[Boolean]
+    ) (ContentAccess.apply _)
+}
+
+class MembersDataApiService(wsClient: WSClient, config: conf.IdentityConfiguration)(implicit executionContext: ExecutionContext) extends Logging {
+
+  // TODO is there an alternative to converting to WSCookie?
+  private def toWSCookie(c: Cookie): WSCookie = {
```

Comment:
Is there an alternative to converting a Play Cookie to a WSCookie here?

Reply:
Mixing `WSCookieConverter` trait gives [`asCookie(c: Cookie): WSCookie`](https://github.com/playframework/play-ws/blob/2214d027f18be8ab77e7778afd91caf7fec7f07b/play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/CookieBuilder.scala#L41)

Reply:
I tried using the `WSCookieConverter` but it converts from  `play.api.libs.ws.ahc.Cookie` to `WSCookie` and I need to go from `play.api.mvc.Cookie` to `WSCookie`. I unfortunately could not find an equivalent converter in the `play.api.mvc package.` Sticking with the solution I had originally in this case.